hub/apps/get-started/dev-options.md

```diff
@@ -17,20 +17,21 @@ There is a wide range of options for developing applications for Windows. The be
 
 To read more about each of these Windows app development options, see [Writing apps for Windows](index.md).
 
-| Feature | .NET MAUI | React Native (RNW) | UWP | Win32 | Windows Forms | WinUI 3 | WPF |
+| Feature | .NET MAUI | React Native (RNW) | UWP XAML (Windows.UI.Xaml) | Win32 | Windows Forms | WinUI 3 | WPF |
 | --- | --- | --- | --- | --- | --- | --- | --- |
 | **Language** | C# | JavaScript, TypeScript | C#, C++, Visual Basic | C++, Rust | C#, Visual Basic | C#, C++ | C#, Visual Basic |
 | **UI language** | XAML/Code | JSX | XAML | Code | Code | XAML | XAML |
 | **UI designer**<br/>(drag & drop) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ | ✅ |
 | **UI debugging** | [Hot Reload](/dotnet/maui/xaml/hot-reload) | [Fast Refresh](https://reactnative.dev/docs/fast-refresh) | [Hot Reload](/visualstudio/xaml-tools/xaml-hot-reload) | - | [Hot Reload](/visualstudio/debugger/hot-reload) | [Hot Reload](/visualstudio/xaml-tools/xaml-hot-reload) | [Hot Reload](/visualstudio/xaml-tools/xaml-hot-reload) |
-| **Fluent Design** | ✅ | ✅ | ❌ | ❌ | ❌ | ✅ | ❌ |
-| **.NET Runtime** | .NET | N/A | Minimal CLR runtime | N/A | .NET & .NET Framework | .NET | .NET & .NET Framework |
+| **Fluent Design** | ✅ | ✅ | ✅ (through [WinUI 2](/windows/apps/winui/winui2)) | ❌ | ❌ | ✅ | ❌ |
+| **.NET Runtime** | .NET | N/A | .NET Core & .NET Native | N/A | .NET & .NET Framework | .NET | .NET & .NET Framework |
 | **Windows App SDK** | ✅ ([more info](/dotnet/maui/platform-integration/invoke-platform-code)) | ✅ ([more info](https://techcommunity.microsoft.com/t5/modern-work-app-consult-blog/getting-started-with-react-native-for-windows/ba-p/912093)) | ❌ | ✅ | ✅ ([more info](../windows-app-sdk/winforms-plus-winappsdk.md)) | ✅ | ✅ ([more info](../windows-app-sdk/wpf-plus-winappsdk.md)) |
-| **Enterprise Apps** | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐ | ⭐⭐⭐⭐⭐ |
+| **Enterprise Apps** | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐ | ⭐⭐⭐⭐⭐ |
 | **Consumer Apps** | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐ | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | ⭐⭐⭐⭐ |
 | **Great for touch** | ✅ | ✅ | ✅ | ❌ | ❌ | ✅ | ❌ |
-| **Cross-platform** | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| **Xbox/HoloLens apps** | ❌ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| **Cross-platform** | ✅ | ✅ | ✅ (through [Uno Platform](/windows/apps/how-tos/uno-multiplatform)) | ❌ | ❌ | ✅ (through [Uno Platform](/windows/apps/how-tos/uno-multiplatform)) | ❌ |
+| **Xbox/HoloLens apps** | ❌ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ |
+| **Sandboxing** | ❌ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ |
```

Comment:
The team doesn't want to add this row here. It's something that can be confused with other types of security options for app deployment. For example, the ability to run most apps in a lightweight appContainer if they're MSIX packaged and deployed with TrustLevel set to appContainer: https://learn.microsoft.com/windows/msix/desktop/desktop-to-uwp-behind-the-scenes#types-of-desktop-app

Reply:
Should it be changed to "Sandboxing (AppContainer)" instead?
WinUI 3 (and other UI frameworks) don't officially support `TrustLevel="appContainer"` yet, you would get crashes on it, so I think it makes sense to add this row to declare that it's only officially supported for UWP to prevent devs from using it on other UI frameworks and getting crashes as a result.

Reply:
We could try that and see how it's received. I think that makes it a bit clearer.